internal/node/proxy.go

```diff
@@ -35,8 +36,8 @@ type ProxyBus struct {
 }
 
 func NewProxyBus(endpoint string, twinID uint32, sub subi.SubstrateExt, signer substrate.Identity, verifyReply bool) (*ProxyBus, error) {
-	if len(endpoint) != 0 && endpoint[len(endpoint)-1] == '/' {
-		endpoint = endpoint[:len(endpoint)-1]
+	if len(endpoint) != 0 {
```

Comment:
i think the check for endpoint length here isn't needed `TrimSuffix` will return an empty string in that case